jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/ListTests.java

```diff
@@ -0,0 +1,43 @@
+package io.github.epam.angular.tests.elements.complex;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.angularPage;
+import static io.github.com.pages.AngularPage.listSection;
+import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+
+public class ListTests extends TestsInit {
+
```

Comment:
Probably also worth thinking about checking the icons?

Reply:
@gbuh , all edits made